tests/models/S3BucketServiceMock.ts

```diff
@@ -0,0 +1,113 @@
+/* istanbul ignore file */
+
+import S3, {Metadata} from "aws-sdk/clients/s3";
```

Comment:
We worked really hard to remove most of these horrible mocks. Is there a genuine reason to put this back?

Reply:
Just to be able to do some unit testing. I tried to follow the model from cert-gen. If you don't want tests for the S3 service I can remove those

Reply:
This can be tested without the mock class - requires use of jest mocks, but much more transparent what the tests are doing. This mock class does "magic" things which make it hard to understand what is being tested and how.